project-base/storefront/components/Pages/Search/SearchContent.tsx

```diff
@@ -76,69 +37,27 @@ export const SearchContent: FC<SearchContentProps> = ({ searchResults, fetching
 
-                        {searchResults.brandSearch.length > 0 && (
+                        {!!searchResults.brandSearch.length && (
                             <div className="mt-6">
                                 <Heading type="h3">{t('Found brands')}</Heading>
-                                <SearchResultsBlock areAllResultsVisible={areBrandsResultsVisible}>
-                                    <SimpleNavigation listedItems={searchResults.brandSearch} />
-                                </SearchResultsBlock>
-                                {numberOfVisible < searchResults.brandSearch.length && (
-                                    <ShowResultsButtonWrapper>
-                                        <Button
-                                            size="small"
-                                            onClick={() => {
-                                                setBrandsResultsVisibility((currentState) => !currentState);
-                                            }}
-                                        >
-                                            {areBrandsResultsVisible ? t('Hide results') : t('Show all results')}
-                                        </Button>
-                                    </ShowResultsButtonWrapper>
-                                )}
+                                <SimpleNavigation listedItems={searchResults.brandSearch} />
                             </div>
                         )}
 
-                        {searchResults.categoriesSearch.totalCount > 0 && !!mappedCategoriesSearchResults && (
+                        {!!searchResults.categoriesSearch.totalCount && !!mappedCategoriesSearchResults && (
                             <div className="mt-6">
                                 <Heading type="h3">{t('Found categories')}</Heading>
-                                <SearchResultsBlock areAllResultsVisible={areCategoriesResultsVisible}>
-                                    <SimpleNavigation listedItems={mappedCategoriesSearchResults} />
-                                </SearchResultsBlock>
-                                {numberOfVisible < mappedCategoriesSearchResults.length && (
-                                    <ShowResultsButtonWrapper>
-                                        <Button
-                                            size="small"
-                                            onClick={() => {
-                                                setCategoriesResultsVisibility((currentState) => !currentState);
-                                            }}
-                                        >
-                                            {areCategoriesResultsVisible ? t('Hide results') : t('Show all results')}
-                                        </Button>
-                                    </ShowResultsButtonWrapper>
-                                )}
+                                <SimpleNavigation listedItems={mappedCategoriesSearchResults} />
```

Comment:
We removed the option to collapse/expand? Why? 🤔

Reply:
Because it is using the SimpleNavigation which on mobile displays slider. It's unified with Category layout.

Reply:
Okay. I will just post a comment about this on the issue, as now we can see 50 categories if there are this many in the results...